### PR TITLE
chore: fix patch linter

### PR DESCRIPTION
**Pull request by nornagon**
#### Description of Change
Patch linter wasn't running correctly when patches were changed, and also the regex wasn't correct.

#### Checklist

- [ ] PR description included and stakeholders cc'd
- [ ] `npm test` passes
- [ ] tests are [changed or added](https://github.com/electron/electron/blob/master/docs/development/testing.md)
- [ ] relevant documentation is changed or added
- [ ] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)
- [ ] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes: none

### CODE REVIEW

**Comment by nornagon** [comment]
cc @zcbenz, can you provide some context for these patches?

patches/chromium/render_widget_host_view_mac.patch
patches/chromium/web_contents.patch
patches/chromium/webview_cross_drag.patch

I'm not exactly sure what they do.

**Comment by codebytere** [comment]
Looks like three missing but otherwise ✅ 
```
Patch file 'patches/chromium/render_widget_host_view_mac.patch' has no description. Every patch must contain a justification for why the patch exists and the plan for its removal.
Patch file 'patches/chromium/web_contents.patch' has no description. Every patch must contain a justification for why the patch exists and the plan for its removal.
Patch file 'patches/chromium/webview_cross_drag.patch' has no description. Every patch must contain a justification for why the patch exists and the plan for its removal.
```

**Comment by nornagon** [comment]
yes, they're missing because i don't know what they do, which is why i tagged @zcbenz :)

File: patches/v8/dcheck.patch [hunk 1]
```diff
@@ -3,6 +3,7 @@ From: Ales Pergl <alpergl@microsoft.com>
 Date: Mon, 22 Oct 2018 10:47:12 -0700
 Subject: dcheck.patch
 
+https://github.com/auchenberg/volkswagen
```
Comment (deepak1556) [inline patches/v8/dcheck.patch]: Wait what 😆

Comment (nornagon) [reply patches/v8/dcheck.patch]: DCHECKs failing? No problem. Here's one easy trick your compilers don't want you to know.

**Comment by zcbenz** [comment]
> patches/chromium/render_widget_host_view_mac.patch

1. Allows Electron to override `acceptsFirstMouse` and `disableAutoHideCursor` for `NSWindow`.
2. Remove private API for MAS build.

(It seems that changes to `render_widget_host_view_mac.mm` is no longer needed in this patch.)

> patches/chromium/web_contents.patch

Allows us to override the `RenderViewHostDelegateView` of WebContents.
https://github.com/electron/libchromiumcontent/pull/226

> patches/chromium/webview_cross_drag.patch

Fix regression of drag-n-drop between different webviews.
https://github.com/electron/libchromiumcontent/pull/267

File: patches/chromium/render_widget_host_view_base.patch [hunk 2]
```diff
@@ -3,6 +3,7 @@ From: Anonymous <anonymous@electronjs.org>
 Date: Thu, 20 Sep 2018 17:46:21 -0700
 Subject: render_widget_host_view_base.patch
 
+... something to do with OSR? and maybe <webview> as well? terrifying.
```
Comment (ckerr) [inline patches/chromium/render_widget_host_view_base.patch]: :100:

**Comment by nornagon** [comment]
CI must be flake as this has 0 behavior impact.

**Pull request merged**